day6_2018.py

```diff
@@ -0,0 +1,145 @@
+        min_y not in x_coords_of_the_nearest_points
+        and max_y not in x_coords_of_the_nearest_points
+        and min_x not in y_coords_of_the_nearest_points
+        and max_x not in y_coords_of_the_nearest_points
+    ):
+        finest_point.append((x_coords[distances_index], y_coords[distances_index]))
+    return closest_points, finest_point
+
+
+def get_all_numbers_of_closest_points(
+    all_distances, x_coords, y_coords, all_x_coords, all_y_coords
+):
+    numbers_of_closest_points = []
+    finite_coords = []
+    for distances_index in range(len(all_distances)):
+        closest_point, finite_coord = get_number_of_closest_points_and_check_if_coord_is_finite(
+            all_distances,
+            all_x_coords,
+            all_y_coords,
+            all_distances[distances_index],
+            x_coords,
+            y_coords,
+            distances_index,
+        )
+        numbers_of_closest_points.append(closest_point)
+        finite_coords.extend(finite_coord)
+    return numbers_of_closest_points, finite_coords
+
+
+def zip_coords_with_numbers_of_closest_points(coords, all_numbers_of_closest_points):
+    coords_and_number_of_points = []
+    for idx, coords in enumerate(coords):
+        coords_and_number_of_points.append([coords, all_numbers_of_closest_points[idx]])
+    return coords_and_number_of_points
+
+
+def get_max_finite_coords_numbers_of_closest_points(
+    zipped_coords_with_numbers_of_closest_points, finite_coords
+):
+    finite_coords_numbers_of_closest_points = []
+    for (
+        coords,
+        numbers_of_closest_points,
+    ) in zipped_coords_with_numbers_of_closest_points:
+        if coords in finite_coords:
+            finite_coords_numbers_of_closest_points.append(numbers_of_closest_points)
+    return max(finite_coords_numbers_of_closest_points)
+
+
+def get_answer_for_part_1(inputs):
+    x_coords, y_coords = get_coords_from_input(inputs)
+    all_distances, all_x_coords, all_y_coords = get_distance_to_all_points(
+        x_coords, y_coords
+    )
+    all_numbers_of_closest_points, finite_coords = get_all_numbers_of_closest_points(
+        all_distances, x_coords, y_coords, all_x_coords, all_y_coords
+    )
+    zipped_coords_with_numbers_of_closest_points = zip_coords_with_numbers_of_closest_points(
+        zip(x_coords, y_coords), all_numbers_of_closest_points
+    )
+    return f"Part 1: {get_max_finite_coords_numbers_of_closest_points(zipped_coords_with_numbers_of_closest_points, finite_coords)} "
+
+
+def get_distance_to_all_coords(x_coords, y_coords):
+    all_distances = []
+    for row_idx in range(max(x_coords)):
+        for column_idx in range(max(y_coords)):
+            distance = []
+            for x_coord, y_coord in zip(x_coords, y_coords):
+                distance.append(
+                    manhattan_distance(row_idx, column_idx, x_coord, y_coord)
+                )
+            if sum(distance) < 10000:
+                all_distances.append(sum(distance))
```

Comment:
```suggestion
            sum_of_distances = sum(distance)
            if sum_of_distances < 10000:
                all_distances.append(sum_of_distances)
```

Jak przypiszesz tę operację do zmiennej, to wykona się ona tylko raz, a później jej wynikiem będziesz tylko operował. A tak to 2 razy liczy się to samo.